boards/arm/weact_stm32g431_core/Kconfig.defconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Andreas Sandberg
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_WEACT_STM32G431_CORE
+
+config BOARD
+	default "weact_stm32g431_core"
+
+# lptim1 is clocked by LSE, set tick freq to 4096 for tick accuracy
+config SYS_CLOCK_TICKS_PER_SEC
+	default 4096 if STM32_LPTIM_TIMER
+
+endif
```

Comment:
Please remove, see https://github.com/zephyrproject-rtos/zephyr/pull/64067.
Though, you'll need to add:
```
# LSE defined as LPTIM clock source by the DTS
CONFIG_STM32_LPTIM_CLOCK_LSE=y
```

Reply:
Done.